cmd/delete.go

```diff
@@ -0,0 +1,80 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/railwayapp/cli/entity"
+	"github.com/railwayapp/cli/ui"
+	"github.com/railwayapp/cli/uuid"
+)
+
+func (h *Handler) Delete(ctx context.Context, req *entity.CommandRequest) error {
+	if len(req.Args) > 0 {
+		// projectID provided as argument
+		arg := req.Args[0]
+
+		if uuid.IsValidUUID(arg) {
+			project, err := h.ctrl.GetProject(ctx, arg)
+			if err != nil {
+				return err
+			}
+
+			return h.ctrl.DeleteProject(ctx, project.Id)
+		}
+
+		project, err := h.ctrl.GetProjectByName(ctx, arg)
+		if err != nil {
+			return err
+		}
+
+		return h.ctrl.DeleteProject(ctx, project.Id)
+	}
+
+	isLoggedIn, err := h.ctrl.IsLoggedIn(ctx)
+	if err != nil {
+		return err
+	}
+
+	if isLoggedIn {
+		return h.deleteFromAccount(ctx, req)
+	} else {
+		return h.deleteFromID(ctx, req)
+	}
```

Comment:
Don't need the else given this is the last function

```suggestion
	return h.deleteFromID(ctx, req)
```